packages/tasit-action/src/contract/Contract.js

```diff
@@ -51,6 +51,11 @@ export class Contract extends Subscription {
     this.#addFunctionsToContract();
   };
 
+  getWallet = () => {
+    const { signer: wallet } = this.#ethersContract;
+    return wallet;
```

Comment:
Is it a "wallet" or an "account"? I think it's only a "wallet" when an "account" or "contract" is wrapped in an app.

Reply:
I thought that since the `tasit-account` is creating an `ethers.js Wallet` we should use this name for now, and when we wrap that with an `Account` object we'll name it as `account`.

Refs:
```javascript
  // Note: For now, `tasit-account` creates a ethers.js wallet object
  // If that changes, maybe this method could be renamed to setAccount()
  setWallet = wallet => {
```

Reply:
That's a fair point.

I might dispute ethers.js calling it a "wallet", but I think when an account is wrapped in even a js object it's already something disctinct, so maybe it is a wallet at that point by my own definition.

Okay, let's keep this as-is

Reply:
Hmmm, although Etherum account wrapped in ethers.js wallet wrapped in Tasit "account" is probably too confusing.

Reply:
Should we keep calling (the future implementation) as tasit account?

Reply:
I think so. We don't want people to think of it as a "wallet" per se from a product perspective

Reply:
Related to https://github.com/tasitlabs/TasitSDK/issues/118
Could we close this thread?

Reply:
Yep, it is somewhat related to that.

Potentially before we close this thread, should we rename the setWallet and getWallet functions to setAccount and getAccount?

Or maybe `setSenderAddress` and `getSenderAddress` in case we want to set it to a contract-based account?

Reply:
Oh, ok sure. I've thought that we'll want to rename it when the `Account` be !== `ethers.Wallet`.
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/374

Reply:
Ah ok. Nah, I think we want to pick our own names that we think are best even when we're using something directly from ethers.

Reply:
Resolving this